configuration/configuration.go

```diff
@@ -685,17 +717,39 @@ func (c *ConfigurationData) GetCacheControlUser() string {
 }
 
 // GetDeprecatedServiceAccountPrivateKey returns the deprecated service account private key (if any) and its ID
-// that is used to verify the service account authentication tokens during key rotation.
+// that is used to verify service account authentication tokens during key rotation.
 func (c *ConfigurationData) GetDeprecatedServiceAccountPrivateKey() ([]byte, string) {
 	return []byte(c.v.GetString(varServiceAccountPrivateKeyDeprecated)), c.v.GetString(varServiceAccountPrivateKeyIDDeprecated)
 }
 
 // GetServiceAccountPrivateKey returns the service account private key and its ID
-// that is used to sign the service account authentication tokens.
+// that is used to sign service account authentication tokens.
 func (c *ConfigurationData) GetServiceAccountPrivateKey() ([]byte, string) {
 	return []byte(c.v.GetString(varServiceAccountPrivateKey)), c.v.GetString(varServiceAccountPrivateKeyID)
 }
 
+// GetDeprecatedUserAccountPrivateKey returns the deprecated user account private key (if any) and its ID
+// that is used to verify user access and refresh tokens during key rotation.
+func (c *ConfigurationData) GetDeprecatedUserAccountPrivateKey() ([]byte, string) {
```

Comment:
Are we gonna deprecate the current key pair when we switch over from kc to local token generation ?